tests/test_attributions_llama.py

```diff
@@ -67,7 +66,7 @@ def verify_small_llama_model(s: torch.Tensor):
         "attn_types": None,
         "init_mode": "gpt2",
         "normalization_type": "RMSPre",
-        "device": device(type="cuda"),
+        "device": DEVICE,
```

Comment:
There is an utils function you could use

Reply:
oops didn't realize that, just updated the tests to use this